### ISSUE TITLE
Should the approval process take other authors more into account

### ISSUE THREAD

**Issue by maelle** [ropensci/dev_guide#466]
At the moment "other authors" are invited to Slack at the same point as authors, but after approval, there's nothing automatic reg their access to the repository.

Should this change? E.g. should they get an invitation to the _organization_ after approval, and then the main author would handle the access to the repo?

**Comment by mpadge** [comment]
Yes, something like that definitely should happen. But it'll be tricky to  accurately identify "other authors", and i can imagine we may well end up needing some kind of explicit input from "main" author. At least potentially asking them to list all those whom they think *should* be added to org prior to final approval step?

**Comment by maelle** [comment]
I mean authors listed in https://github.com/ropensci/software-review/blob/983614e3d70a6565fcb0030cd295fc715966158a/.github/ISSUE_TEMPLATE/A-submit-software-for-review.md?plain=1#L13

Now even if we invite them to the organization I don't think we should invite them to the package team since it has admin access and that's not necessarily what the maintainer wants.

**Comment by mpadge** [comment]
Yes, but also what i meant is that submitting authors are also free to mention *all* other authors of pkg, including "cph" authors and the like who might not have made any active contribution to the pacakge itself. So even that list out not necessarily be judged as "reliable". Does that make sense?

**Comment by maelle** [comment]
in that case should we edit the HTML comment description? Since we also invite them to slack, we assume they're "active".

**Comment by maelle** [comment]
@mpadge it seems we have an unresolved discussion above. :smile_cat:

**Comment by maelle** [comment]
TODO: rename field to "other main contributors"

**Comment by maelle** [comment]
To me it seems better to let maintainers ask us, and anyway I feel we're moving towards fewer transfers.
